apps/ledger-live-mobile/ios/Podfile

```diff
@@ -8,6 +8,7 @@ require Pod::Executable.execute_command('node', ['-p',
 
 
 platform :ios, '13.4'
+use_modular_headers!
```

Comment:
We already setup `use_frameworks! :linkage => linkage.to_sym` below that will be static or dynamic depending on env vars
So I don't think you need to add anything here, as seen here too: https://stackoverflow.com/a/53265888